app/components/Session/SavedSessions.js

```diff
@@ -1,56 +1,88 @@
 import React, { Component } from 'react';
 import moment from 'moment';
-import { Button, Row, Col } from 'antd';
+import { Button, Row, Col, Table } from 'antd';
 import FormattedCaps from './FormattedCaps';
 import SessionCSS from '../Session.css';
 
 
 export default class SavedSessions extends Component {
 
-  handleDelete (index) {
+  handleDelete (uuid) {
     return () => {
       if (window.confirm('Are you sure?')) {
-        this.props.deleteSavedSession(index);
+        this.props.deleteSavedSession(uuid);
       }
     };
   }
 
-
+  sessionFromUUID (uuid) {
+    const {savedSessions} = this.props;
+    for (let session of savedSessions) {
+      if (session.uuid === uuid) {
+        return session;
+      }
+    }
+    throw new Error(`Couldn't find session with uuid ${uuid}`);
+  }
 
   render () {
     const {savedSessions, setCaps, capsUUID, switchTabs} = this.props;
 
-    return (<Row>
+    const columns = [{
+      title: 'Capability Set',
+      dataIndex: 'name',
+      key: 'name'
+    }, {
+      title: 'Created',
+      dataIndex: 'date',
+      key: 'date'
+    }, {
+      title: 'Actions',
+      key: 'action',
+      render: (text, record) => {
+        let session = this.sessionFromUUID(record.key);
+        return (
+          <div>
+            <Button icon='edit'
+             onClick={() => {setCaps(session.caps, session.uuid); switchTabs('new');}}
+             className={SessionCSS['edit-session']}
+            />
+            <Button icon='delete' onClick={this.handleDelete(session.uuid)}/>
+          </div>
+        );
+      }
+    }];
+
+    let dataSource = [];
+    if (savedSessions) {
+      dataSource = savedSessions.map((session) => {
+        return {
+          key: session.uuid,
+          name: (session.name || '(Unnamed)'),
+          date: moment(session.date).format('YYYY-MM-DD')
+        };
+      });
+    }
+
+    let onRowClick = (record) => {
+      let session = this.sessionFromUUID(record.key);
+      setCaps(session.caps, session.uuid);
+    };
+
```

Comment:
I see what you're saying. I usually bind to `this` in the calling location, but I see your point that you could bind it in the constructor instead.